### ISSUE TITLE
Internal Server Error when trying to Edit on Linux hosted version

### ISSUE THREAD

**Issue by LukasHolzner** [toohoo/thofaq#15]
I installed the FAQ on a Linux (Debian 10 Buster) machine with apache2. Unfortunately, when I try to edit the categories I get an Internal Server Error.
The error.log of apache has following lines resulting of the call:
```
[Fri Nov 08 08:50:07.773874 2019] [cgid:error] [pid 6573:tid 139900639544448] (2)No such file or directory: AH01241: exec of '/var/www/html/faq/cgi-bin/editfaq.pl' failed
[Fri Nov 08 08:50:07.774279 2019] [cgid:error] [pid 815:tid 139900078769920] [client ::1:46094] End of script output before headers: editfaq.pl, referer: http://localhost/faq/cgi-bin/faq.pl?kat=1
```
Do you have any idea how to fix this?

**Comment by toohoo** [comment]
OK, I've set up an Linux (debian) VM under Win to test it. Seems my configuration is corrupt.
Hint for further testing. You also can try to find a more detailed message with a call of Perl and the script. Navigate to the directory, let's say `/var/www/html/thofaq/cgi-bin` and call perl with the affected script - let's say for example:
```
xxx@thofaq:/var/www/html/thofaq/cgi-bin# perl -w editfaq.pl
```
.. where "**-w**" is telling warnings.
I did add an break of the script with `exit` but nonetheless an error arises. If I run it with perl all is fine.
The message "_No such file or directory_" is related to line:
```
require "thpl.pl";
```
.. which is not found in the PATH. This is disturbing because it is located in the same directory as the script itself.
This is the status I did come until now.
